cmd/kubeadm/app/apis/output/types.go

```diff
@@ -82,3 +82,38 @@ type UpgradePlan struct {
 
 	ConfigVersions []ComponentConfigVersionState
 }
+
+// Certificate represents information for a certificate or a certificate authority when using the check-expiration command.
+type Certificate struct {
+	// Name of the certificate.
+	Name string
+
+	// ExpirationDate defines certificate expiration date.
```

Comment:
```suggestion
	// ExpirationDate defines certificate expiration date in UTC following the RFC3339 format.
```

please update this in the private v1alpha3 type too.